dod-advana/gamechanger-web
# Task/UOT-132472 - Graph Re-center Behavior

## Description


- Fixes graph re-center not working after clicking "Load All" or "Reset graph"
- Adjusts force parameters to reduce nodes overlapping each other and make unconnected nodes more local (this should have a minor performance benefit as well)
- Fix eslint warnings

## !vibez


😎

## Related Issue/Ticket



https://jira.di2e.net/browse/UOT-132472

## Types of changes

## Smoke testing instructions



Graph view - after clicking "Load All" button (pull up search with #results > 500)
1. Wait for graph to "settle"
2. View should zoom to the center connected graph

Repeat for reset graph button

![image](https://user-images.githubusercontent.com/85301886/156633035-3e8e5010-8277-4bf9-8aac-a248bbdcd89f.png)


## Checklist:
